### PR TITLE
mesh: update sidebar to follow IA work for install

### PR DESCRIPTION
**Pull request by lahabana**
### Description

The install page was using older architecture which is not the case anymore according to the IA change.

### Testing instructions

Netlify link: 

### Checklist 

- [ ] Review label added 
- [ ] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_data/docs_nav_mesh_2.2.x.yml [hunk 1]
```diff
@@ -133,35 +139,6 @@ items:
...
-        url: /installation/redhat/
-      - text: CentOS
-        url: /installation/centos/
-      - text: Debian
-        url: /installation/debian/
-      - text: Ubuntu
-        url: /installation/ubuntu/
-      - text: macOS
-        url: /installation/macos/
-      - text: Windows
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_mesh_2.2.x.yml:163]: @lahabana Do we need to keep Windows since that's Kong Mesh-specific?

Comment (lahabana) [reply app/_data/docs_nav_mesh_2.2.x.yml:163]: How was this supposed to fit in IA?

Comment (cloudjumpercat) [reply app/_data/docs_nav_mesh_2.2.x.yml:163]: @lahabana I think the "Install kumactl" and "Use Kuma" pages were meant to replace all the install pages, but once I got to the Mesh ones, I remembered that those pages weren't shared with the Kuma docs. They were separate. If there isn't a reason why they should be different/their own files, I think we can mirror what's currently in the Kuma nav and remove all the install topics and replace them with "Install kumactl" and "Use Kuma". 

Looking at the Windows one, I think we'd miss some of the Windows specifics in that file, but I think [Install kumactl](https://kuma.io/docs/2.2.x/production/install-kumactl/) does cover the general things like adding kumactl to your path.

Comment (lahabana) [reply app/_data/docs_nav_mesh_2.2.x.yml:163]: Ok I've added back a windows install page which seems to work. I've also did minor fixes to remove warnings

**Review by cloudjumpercat — APPROVED** [review]
Everything LGTM! Thanks for making these updates!

**Pull request merged**